torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5464,6 +5473,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            # Batched grad checks fail for empty input tensors (see https://github.com/pytorch/pytorch/issues/53407)
            check_batched_grad=False,
            check_batched_gradgrad=False,
+           supports_forward_ad=True,
```

Comment:
Random question but... what happens if someone tries to use a dual_tensor with an out= operation?

Reply:
Very good point. It silently ignores it for now I think.

Reply:
Actually it is only missing the check for the "out" argument. All the other ones are properly handled.

Reply:
Added a PR below this one to fix this.